src/v2/modules/contacts/validator.js

```diff
@@ -2,8 +2,10 @@
 
 const Joi = require('@hapi/joi');
 const validators = require('../../lib/validators');
+const contactTypes = require('../../lib/contact-types');
 
 const personSchema = Joi.object({
+  title: validators.OPTIONAL_STRING,
```

Comment:
I don't think we should probably rename salutation to title on the way in here, as presumably in other cases when the CRM responds with a contact it will come out as 'salutation' still so isn't consistent.

If we want to refer to it as 'title' in the water service, the only place it needs to live is the water service contact mapper.